client/src/css/Products.css

```diff
@@ -1,57 +1,98 @@
+
 #product{
-    width: 100%;
-    display: flex;
-    justify-content: space-around;
-    flex-wrap: wrap;
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  flex-wrap: wrap;
 }
 .card{
-    min-width: 300px;
-    min-height: 400px;
-    border: 1px solid #eee;
-    overflow: hidden;
-    padding: 10px;
-    box-shadow: 2px 8px 20px #ddd;
-    margin: 10px;
-    transition: 0.5s linear;
+  min-width: 300px;
+  min-height: 400px;
+  border: 1px solid #eee;
+  overflow: hidden;
+  padding: 10px;
+  box-shadow: 2px 8px 20px #ddd;
+  margin: 10px;
+  transition: 0.5s linear;
 }
 .card:hover{
-    box-shadow: none;
+  box-shadow: none;
 }
 .card img{
-    max-width: 300px;
-    width: 100%;
-    height: 100%;
-    max-height: 300px;
-    display: block;
-    object-fit: cover;
+  max-width: 300px;
+  width: 100%;
+  height: 100%;
+  max-height: 300px;
+  display: block;
+  object-fit: cover;
 }
 .card h3{
-    text-transform: uppercase;
-    margin: 10px 0;
+  text-transform: uppercase;
+  margin: 10px 0;
 }
 .card h3 a{
-    text-decoration: none;
-    color: #333;
+  text-decoration: none;
+  color: #333;
 }
 .card h3 a:hover{
-    color: crimson;
+  color: crimson;
 }
 .card span{
-    color: crimson;
+  color: crimson;
 }
 .card p{
-    margin: 10px 0;
+  margin: 10px 0;
 }
 .card button{
-    border: none;
-    outline: none;
-    background: #333;
-    color: white;
-    width: 100%;
-    height: 40px;
-    display: block;
-    cursor: pointer;
-    text-transform: uppercase;
-    letter-spacing: 2px;
-    margin: 15px 0;
+  border: none;
+  outline: none;
+  background: #333;
+  color: white;
+  width: 100%;
+  height: 40px;
+  display: block;
+  cursor: pointer;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+  margin: 15px 0;
+}
+
+
+
+
```

Comment:
Empty lines. Code should be consistent. I mean written in the same styling. So if in one place we use couple empty lines then why in another similar place of code we do not use them? We should stick to the same approach and either use empty lines if they needed either do not use them in the similar places of code